index.html

```diff
@@ -112,18 +131,23 @@ <h3 class="project-name">Tonic</h3>
 				</div>
 			</div>
 			<div class="work-section__card">
-				<div class="work-section__card--img"><img src="/images/SnapshootPortfolio(3).png" alt=""></div>
+				<div class="work-section__card--img mobile"><img src="/images/SnapshootPortfolio(3).png" alt=""></div>
+				<div class="work-section__card-img desktop order--two"><img src="./images/Card-images/SnapshootPortfolioDesktop4.png" alt="card snap"></div>
 				<div class="work-section__card--info">
-					<h3 class="project-name">Multi-post Stories</h3>
+					<h3 class="project-name mobile">Multi-post Stories</h3>
+					<h3 class="project-name desktop">Uber Navigation</h3>
 					<div class="range">
-						<p class="range__canopy">CANOPY</p>
+						<p class="range__canopy mobile">CANOPY</p>
+						<p class="range__canopy desktop">UBER</p>
 						<div class="range__circle"></div>
-						<p class="range__txt">Back End Dev</p>
+						<p class="range__txt mobile">Back End Dev</p>
+						<p class="range__txt desktop">Lead Dev</p>
 						<div class="range__circle"></div>
-						<p class="range__txt">2015</p>
+						<p class="range__txt mobile">2015</p>
+						<p class="range__txt desktop">2018</p>
 					</div>
 					<p class="general__desc">
-						A daily selection of privately personalized reads; no accounts or sign-ups required.
+						A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
```

Comment:
- [ ] Please, kindly correct these lines, so they look like the template (you can use padding, etc):

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/149452436-d57e875e-f9d5-4807-93a4-125a73698c0c.png) | ![image](https://user-images.githubusercontent.com/57421823/149452461-3f1c81e5-5e92-4b6b-b164-e0cb23552880.png)

Let's do this so your project becomes more striking! 👍